index.html

```diff
@@ -149,6 +149,18 @@ <h3>SEE THE WHOLE PROGRAM</h3>
             </div>
         </section>
 
+        <section class="featured-speakers-heading">
+            <div class="featured-speakers-heading">
+                <h5>Featured Speakers</h5>
+                <div id="line"></div>
+            </div>
+            <div id="featured-speakers"></div>
+            <div class="more mobile">
+                <p>MORE</p>
+                <img src="imgs/more-arrow.png">
+            </div>
+        </section>
+
```

Comment:
Hello @AbdusSattar-70, the feautured speakers section isn't showing on my live demo link because I only deployed the main branch but not this featured-speakers branch which has the featured speakers section.

Reply:
> Hello @AbdusSattar-70, the feautured speakers section isn't showing on my live demo link because I only deployed the main branch but not this featured-speakers branch which has the featured speakers section.

@Omwomo, I appreciate your clarification. I would suggest deploying the complete app and making the necessary changes to the "requires" fields in the readme, as mentioned in the video. Additionally, I wasn't able to directly comment on your readme, index.html, and other files due to the merging process without review.